package.json

```diff
@@ -36,44 +36,45 @@
     "url": "https://github.com/myfarms/mf-select/issues"
   },
   "devDependencies": {
-    "@angular/common": "^6.0.0",
-    "@angular/compiler": "^6.0.0",
-    "@angular/compiler-cli": "^6.0.0",
-    "@angular/core": "^6.0.0",
-    "@angular/forms": "^6.0.0",
-    "@angular/platform-browser": "^6.0.0",
-    "@angular/platform-browser-dynamic": "^6.0.0",
-    "@compodoc/compodoc": "^1.0.0-beta.10",
-    "@types/jest": "^22.0.1",
-    "@types/node": "~6.0.60",
-    "codelyzer": "~4.4.2",
-    "concurrently": "^3.4.0",
-    "core-js": "^2.4.1",
+    "@angular/common": "^9.0.0",
+    "@angular/compiler": "^9.0.0",
+    "@angular/compiler-cli": "^9.0.0",
+    "@angular/core": "^9.0.0",
+    "@angular/forms": "^9.0.0",
+    "@angular/platform-browser": "^9.0.0",
+    "@angular/platform-browser-dynamic": "^9.0.0",
+    "@compodoc/compodoc": "^1.1.11",
+    "@types/jest": "^22.2.3",
+    "@types/node": "^6.0.118",
+    "codelyzer": "^4.4.4",
+    "concurrently": "^3.6.1",
+    "core-js": "^2.6.11",
     "del": "^2.2.2",
     "fs-extra": "^5.0.0",
-    "gulp": "^3.9.1",
-    "gulp-rename": "^1.2.2",
-    "gulp-rollup": "^2.11.0",
-    "jest": "^22.1.4",
-    "jest-preset-angular": "^5.0.0",
-    "lite-server": "^2.3.0",
-    "node-sass": "^4.5.2",
-    "node-sass-tilde-importer": "^1.0.0",
-    "node-watch": "^0.5.2",
+    "gulp": "3.9.1",
+    "gulp-rename": "^1.4.0",
+    "gulp-rollup": "^2.17.0",
+    "jest": "^22.4.4",
+    "jest-preset-angular": "^5.2.3",
+    "lite-server": "^2.5.4",
+    "node-sass": "^4.14.1",
+    "node-sass-tilde-importer": "^1.0.2",
+    "node-watch": "^0.5.9",
     "protractor": "~5.1.2",
     "rollup": "^0.49.3",
     "run-sequence": "^1.2.2",
-    "rxjs": "^6.0.0",
+    "rxjs": "^6.5.5",
     "systemjs": "^0.20.12",
     "ts-node": "~3.2.0",
     "tslint": "~5.7.0",
-    "typescript": "~2.7.2",
-    "zone.js": "^0.8.26"
+    "typescript": "~3.8",
+    "zone.js": "^0.8.29"
   },
   "engines": {
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "angular2-virtual-scroll": "^0.3.2"
+    "angular2-virtual-scroll": "^0.4",
+    "tsickle": "^0.38.1"
```

Comment:
I had to install this because the build process failed without it. Something must require it but not have it listed in a package.json or something. You could remove it and run build to see what I mean.

Reply:
thats weird. even deleting the lock file doesnt fix it so idk whats up with that